### PR TITLE
Arabic translation enhancement: Hide unit count with dual forms.

### PR DESCRIPTION
**Pull request by 6km**
The issue with is described in #221.
I have implemented a solution for it.

- **Adds a property to the Arabic humanizer** called `hideUnitInDualForm` that I used to indicate that the language is Arabic.
image

- **Adds a new option**, which is `arabic_hideUnitInDualForm`, it works only with Arabic language.
*default:* `true`.
image

I could just check if the language equals `ar`, but I thought it would be better to add that `hideUnitInDualForm` property to the humanizer and options!

### CODE REVIEW

**Comment by EvanHahn** [comment]
Thank you! I will take a look soon.

File: humanize-duration.js [hunk 1]
```diff
@@ -186,7 +186,8 @@
       ),
       {
         delimiter: " ﻭ ",
-        _digitReplacements: ["۰", "١", "٢", "٣", "٤", "٥", "٦", "٧", "٨", "٩"]
+		hideUnitInDualForm: true,
```
Comment (EvanHahn) [inline humanize-duration.js:189]: Could you make this a private option, `_hideCountIf2`?

File: humanize-duration.js [hunk 2]
```diff
@@ -1655,10 +1656,21 @@
...
+	/** @type {boolean} */
+	var hideUnitInDualForm = false;
+	if (options.language === "ar") {
+		if ("arabic_hideUnitInDualForm" in options) {
+			hideUnitInDualForm = options.arabic_hideUnitInDualForm;
+		} else if ("hideUnitInDualForm" in language) {
+			hideUnitInDualForm = language.hideUnitInDualForm;
+		}
+	}
+
```
Comment (EvanHahn) [inline humanize-duration.js:1674]: I think you can remove this and just use `options._hideCountIf2`.

**Review by EvanHahn — CHANGES_REQUESTED** [review]
- I don't know much about Arabic. Do you have a place I could learn more about these rules to make sure this is correct for all Arabic speakers?

- Please run `npm format` to format the code.

- It looks like tests are failing. Could you run `npm test` to make the tests pass?

**Comment by 6km** [comment]
@EvanHahn Thanks for your review! I'm working work on it!

> * I don't know much about Arabic. Do you have a place I could learn more about these rules to make sure this is correct for all Arabic speakers?

I didn't find many English resources talking about the dual nouns (Muthanna) in Arabic, but this can help I guess.
[Number: Singular, Dual, Plural Noun in Arabic](https://learnarabicstepbystep.blogspot.com/2018/11/number-singular-dual-plural-arabic-noun.html#:~:text=dual%2C%20or%20plural.-,Dual%20noun,-Dual%20is%20used)

**Comment by EvanHahn** [comment]
Thanks! I'll take another look soon.

**Comment by EvanHahn** [comment]
This was released in `humanize-duration@3.32.1`. Thanks for your contribution!

**Comment by 6km** [comment]
> This was released in `humanize-duration@3.32.1`. Thanks for your contribution!

You're welcome!

**Pull request merged**